frontend/lib/framework/error-classes-list.cpp

```diff
@@ -537,4 +537,11 @@ void ErrorProcDefExplicitAnonFormal::write(ErrorWriterBase& wr) const {
   wr.code(fn, {formal});
 }
 
+void ErrorSuperFromTopLevelModule::write(ErrorWriterBase& wr) const {
+  auto use = std::get<const uast::AstNode*>(info);
+  wr.heading(kind_, type_, use, "cannot use/import super from a toplevel "
+                                "module");
+  wr.code(use, {use});
```

Comment:
Wanna lock this in a .good file somehow? Check out `test/errors` (I think) for examples. It also helps to actually see what the error message looks like "in practice".